include/solvers/gls_sharp_navier_stokes.h

```diff
@@ -67,33 +67,43 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
   // BB - TODO The particles structure should be refactored to use a small class
   // to store the information or a struct instead of just using a vector where
   // the things are hardcoded within.
+
+  // Defined the particle structure and value based on the parameter file.
   void
   define_particles();
 
+  // Evaluate the forces applied on each of the Ib particles.
   void
   force_on_ib();
 
+  // Modified the system matrix to impose IB condition
   void
   sharp_edge();
 
+  // Right in  the ouput file the force , velocity , position of each of the particles
```

Comment:
Be more careful about spelling. This is important...
```suggestion
  // Write in  the ouput file the force , velocity , position of each of the particles
```